### ISSUE TITLE
Trojan - IIO-Oscilloscope - v0.17-master

### ISSUE THREAD

**Issue by Pekiel** [analogdevicesinc/iio-oscilloscope#508]
Hello,

IIO-Oscilloscope - v0.17-master for Windows is detected as Trojan by ESET LiveGuard.
Palo Alto is detecting file as "Virus/Linux.WGeneric.efjyuh" by WildFire.

**Comment by Pekiel** [comment]
Yes. Directly from this site. When I download the file, ESET is imidietly deleting the file.
![image](https://github.com/user-attachments/assets/1f7f4ebf-82bb-4d17-97cf-62fd53a095ef)

**Comment by Pekiel** [comment]
My Firewall (Palo Alto) have the problem with the file for Linux
![image](https://github.com/user-attachments/assets/209093c1-4be3-4682-8950-f06e7fa15f6d)

**Comment by dNechita** [comment]
Thanks for reporting this!
All installers available at: https://github.com/analogdevicesinc/iio-oscilloscope/releases are digitally signed, which should typically reassure antivirus software that the installer is not a threat. However, it seems this is not sufficient in this case. I will investigate further to determine what additional actions are needed to prevent future false positive detections.
If your antivirus is blocking the download, you can add the installer to the whitelist.
